manifests/base/deployment.yaml

```diff
@@ -1,31 +1,30 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: backstage
-  namespace: backstage
+  name: service-catalog
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: backstage
+      app: service-catalog
   template:
     metadata:
       labels:
-        app: backstage
+        app: service-catalog
     spec:
       containers:
-        - name: backstage
-          image: quay.io/skopecky/backstage
+        - name: service-catalog
+          image: service-catalog
           command: ["node"]
           args: []  # added by patch in overlays
           imagePullPolicy: Always
           ports:
             - containerPort: 7007
           envFrom:
             - secretRef:
-                name: hippo-pguser-backstage
+                name: hippo-pguser-service-catalog
```

Comment:
Small nit. Any reason why the db is named `hippo`?

Reply:
Hmm, I think there is no reason for it. I'll change it to `service-catalog` so that it makes more sense.